src/options/views/tab-settings/vm-import.vue

```diff
@@ -127,8 +127,12 @@ async function importBackup(file) {
         },
         position: more.position,
         props: {
-          lastModified: more.lastModified || +entry.lastModDate,
-          lastUpdated: more.lastUpdated || +entry.lastModDate,
+          lastModified: more.lastModified
+            || more.props.lastModified // Import data from Tampermonkey
```

Comment:
`more.props` would not exist when importing from a Violentmonkey export. You can use optional chaining (`?.`) here.